### DISCUSSION TITLE
Tank mode support in future?

### DISCUSSION THREAD

**Discussion by inventor7777** [EdgeTX/edgetx#4566] [Ideas]
Hi there! I have a RadioMaster MT12 and I would like to eventually use it to control a tracked vehicle with a track on either side. I currently use a series of logical functions and overrides to accomplish a rough support for this vehicle. 

Is it possible to add this natively in the future? Thank you!

**Comment by pfeerick** [comment]
I think in order to have any chance of answering that question would need to know exactly what you think that would look like / how it would operate? The answer however may be no if it is possible to do with existing tools, as it really is just a matter of documentation / templates / wizard at that point.

**Comment by kmf123kmf** [comment]
I'm not sure what kind of functionality you're trying to achieve but I was able to get a tank model working pretty quickly on an MT12 using only the mixing features.  It's conceptually the same as elevon mixing for an airplane.  I wasn't able to actually test it as I don't have a tank model, but everything seems to checkout in the channel monitor.  

Assuming CH1 = left track and CH2 = right track
Make 2 inputs, ST and TH

Then add 2 mixes per channel.

```
CH1  100 I TH
 +=  100 I ST
CH2  100 I TH
 += -100 I ST
```
At full throttle one track stays at 100% while the opposing side reduces to steer, but as throttle decreases both tracks start contributing to the steer.  Steering at 0 throttle has both tracks going in opposite directions.

Is this what you're looking for?  If it works, the only thing you may need to play with is the steering weighting to get the proper motion and feel.

**Reply by inventor7777** [reply]
Thank you so much!

I am new to the whole mixing thing and I hadn't figured out how to do that yet! I really appreciate the tip!

**Reply by inventor7777** [reply]
It works perfectly, thanks!
